### DISCUSSION TITLE
[Validator] Option to specify the format for validating dates?

### DISCUSSION THREAD

**Discussion by simondeeley** [symfony/symfony#50649] [Polls]
Currently, the Validator component's "Date" ([link](https://symfony.com/doc/current/reference/constraints/Date.html)) validator only validates inputs with the format `Y-m-d`.  Whilst this is perfectly fine for general international use, I am finding that client-side a lot of dates are inputted in locale-specific formats, e.g. `d-m-Y`.

Is there enough interest to add the option to specify the date format?  Or, going one-stage further, add the ability to specify an array of valid date formats?

**Comment by ghost** [comment]
> No - I think the date validator should stay international and rely on client-side formatting

`Y-m-d` is considered international format? Any spec reference?

**Reply by leberknecht** [reply]
![image](https://github.com/symfony/symfony/assets/1165177/37f083bc-fcb6-4ca4-bd2d-62fab6cffc97)

**Reply by simondeeley** [reply]
Thanks for your thoughts 👍🏻
Just posing the question though that if international formatting wasn't an option, should the validator constraint allow devs the option to override/customise the formatting, e.g. `d-m-Y`, for example?
